packages/form-layout/src/layouts/abstract-layout-controller.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * An abstract class for layout controllers. Not intended for public use.
+ *
+ * @private
+ */
+export class AbstractLayoutController {
+  constructor(host, config) {
```

Comment:
Would it be beneficial to add a JSdoc type definition here to help with autocomplete? 
Something like:
```js
  /**
   * @typedef {Object} LayoutControllerConfig
   * @property {MutationObserverInit} mutationObserverOptions - Options for the MutationObserver.
   */

  /**
   * @param {HTMLElement} host
   * @param {LayoutControllerConfig} config
   */
   constructor(host, config) { ... }
```

Reply:
Done.